MapboxVision/View/DetectionsView.swift

```diff
@@ -77,6 +79,8 @@ private extension DetectionClass {
             return "Person"
         case .bicycle:
             return "Bicycle"
+        case .constructionCone:
+            return "Construction Cone"
```

Comment:
I'd go for something shorter not to overcrowd the screen.
```suggestion
            return "Cone"
```

Reply:
Agree.